crates/sui-graphql-rpc/src/types/protocol_config.rs

```diff
@@ -30,7 +30,9 @@ pub(crate) struct ProtocolConfigFeatureFlag {
 
 #[derive(Clone, Debug)]
 pub(crate) struct ProtocolConfigs {
-    native: NativeProtocolConfig,
+    version: u64,
+    configs: BTreeMap<String, Option<String>>,
```

Comment:
Do we need the `Option` here? (Maybe the rest of the PR will answer this question for me).

Reply:
(Attempting to answer this for myself -- our GraphQL schema differentiates between a config that doesn't exist at any version, and a config that exists but hasn't been set, so that's why we need this `Option`).

Reply:
Close. But it's not due to GraphQL differentiating that - protocol config values themselves are `Option`s, likely due to the reason you mentioned.